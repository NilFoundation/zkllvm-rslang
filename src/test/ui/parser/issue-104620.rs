#![feature(rustc_attrs)]

#![rustc_dummy=5z] //~ ERROR unexpected expression: `5z`
fn main() {}
