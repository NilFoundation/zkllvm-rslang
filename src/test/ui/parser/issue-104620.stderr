error: unexpected expression: `5z`
  --> $DIR/issue-104620.rs:3:16
   |
LL | #![rustc_dummy=5z]
   |                ^^

error: aborting due to previous error

